autogen/code_utils.py

```diff
@@ -242,7 +242,11 @@ def execute_code(
             If a list or a str of image name(s) is provided, the code will be executed in a docker container
             with the first image successfully pulled.
             If None, False or empty, the code will be executed in the current environment.
-            Default is True, which will be converted into a list.
+            Default is None, which will be converted into a Empty list.
```

Comment:
```suggestion
            Default is None, which will be converted into an empty list when docker package is available.
```

Reply:
Thanks for the correction ....Have corrected it in my latest commit [Fixed line 245 FIX_02](https://github.com/microsoft/autogen/pull/233/commits/c0c664d4ba425cc1d7d16b97dee027886e768957)👍